jdi-light/src/main/java/com/epam/jdi/light/settings/TestProperties.java

```diff
@@ -0,0 +1,263 @@
+    testsProperties.put("requests.strategy",
+        new Pair<>(p -> LOGS.requestsStrategy = getLoggerStrategy(p),doNothing));
+    testsProperties.put("browser.kill",
+        new Pair<>(p -> COMMON.killBrowser = p,doNothing));
+    testsProperties.put("element.search.strategy",
+        new Pair<>(TestProperties::setSearchStrategy,doNothing));
+    testsProperties.put("browser.size",
+        new Pair<>(DRIVER.screenSize::read,doNothing));
+    testsProperties.put("page.load.strategy",
+        new Pair<>(p -> DRIVER.pageLoadStrategy = getPageLoadStrategy(p),doNothing));
+    testsProperties.put("page.check.after.open",
+        new Pair<>(p -> PAGE.checkPageOpen = parse(p),doNothing));
+    testsProperties.put("assert.type",
+        new Pair<>(SoftAssert::setAssertType,doNothing));
+    testsProperties.put("click.type",
+        new Pair<>(p -> ELEMENT.clickType = getClickType(p),doNothing));
+    testsProperties.put("text.type",
+        new Pair<>(p -> ELEMENT.getTextType = getTextType(p),doNothing));
+    testsProperties.put("set.text.type",
+        new Pair<>(p -> ELEMENT.setTextType = getSetTextType(p),doNothing));
+//     RemoteWebDriver properties
+    testsProperties.put("remote.type",
+        new Pair<>(p -> DRIVER.remoteUrl = getRemoteUrl(p),doNothing));
+    testsProperties.put("driver.remote.url",
+        new Pair<>(p -> DRIVER.remoteUrl = p,doNothing));
+    testsProperties.put("log.level",
+        new Pair<>(p -> LOGS.logLevel = parseLogLevel(p), () -> logger.setLogLevel(LOGS.logLevel)));
+    testsProperties.put("allure.steps",
+        new Pair<>(p -> LOGS.writeToAllure = parseBoolean(p),doNothing));
+    testsProperties.put("smart.locators",
+        new Pair<>(p -> ELEMENT.smartTemplate = p.split(";")[0],doNothing));
+    testsProperties.put("smart.locators.toName",
+        new Pair<>(p -> ELEMENT.smartName = getSmartSearchFunc(p),doNothing));
+    testsProperties.put("smart.search",
+        new Pair<>(p -> ELEMENT.useSmartSearch = getSmartSearchUse(p),doNothing));
+    testsProperties.put("headless",
+        new Pair<>(p -> DRIVER.capabilities.common.put("headless", p),doNothing));
+  }
+
+  private static Strategies getStrategy(String prop) {
+    String strategy = prop.trim().toLowerCase();
+    switch (strategy) {
+      case "jdi":
+        return JDI;
+      case "selenium":
+        return SELENIUM;
+      default:
+        return JDI;
+    }
+  }
+
+  private static List<Strategy> getLoggerStrategy(String strategy) {
+    if (isBlank(strategy))
+      return new ArrayList<>();
+    List<com.epam.jdi.light.logger.Strategy> strategies = new ArrayList<>();
+    try {
+      String[] split = strategy.split(";");
+      strategies = LinqUtils.map(split, s -> parseStrategy(s.trim()));
+    } catch (Exception ignore) { }
+    return strategies;
+  }
+
+  private static void setSearchStrategy(String p) {
+    p = p.toLowerCase();
+    if (p.equals("soft"))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5524501)